src/redux/actions/loginAction.js

```diff
@@ -0,0 +1,48 @@
+import axios from "axios";
+import $ from "jquery";
+import history from "../../routes/history";
+import {
+  LOGIN_SUCCESSFUL,
+  LOGIN_ERROR,
+  LOGOUT_SUCCESSFUL
+} from "../action_types";
+import homepage from "../../views/Home/index";
+
+const url = "https://ah-magnificent6-staging.herokuapp.com/api/";
+
+export const loginSuccessful = message => ({
+  type: LOGIN_SUCCESSFUL,
+  message
+});
+
+export const loginError = error => ({
+  type: LOGIN_ERROR,
+  error
+});
+
+export const logoutsuccessful = () => ({
+  type: LOGOUT_SUCCESSFUL
+});
+
+export default user => dispatch => {
```

Comment:
Why is this thunk different in format from the one on line 44? 
The format for an action thunk should be 

export const user = ( ) => 
  (dispatch) => {
    ....
  }

When you import this thunk in other files, ensure you import it as user and do not rename it, to avoid confusing others when it comes to tracking down this action as the project grows.
Also, be more specific with your naming, @micahoriaso has a similar thunk, so which is which?